R/with.r

```diff
@@ -128,9 +143,13 @@ set_libpaths <- function(paths) {
   invisible(old)
 }
 
+reset_libpaths <- function(paths) {
```

Comment:
Maybe it would be to handle this consistently with the other setter functions, i.e. add a `add` argument?